models/users.js

```diff
@@ -530,7 +530,11 @@ Users.helpers({
 
   getStartDayOfWeek() {
     const profile = this.profile || {};
-    return profile.startDayOfWeek || 1;
```

Comment:
You can also use the new `??` operator since yesterday, see: https://github.com/tc39/proposal-nullish-coalescing, like so: `return profile.startDayOfWeek ?? 1;`

Reply:
Changed - thanks for the feedback!